components/karma.conf.js

```diff
@@ -42,13 +42,13 @@ module.exports = function(config) {
 
         // TODO: remove this after official support is added
         customLaunchers: {
-            FirefoxHeadless: {
-                base: 'Firefox',
-                flags: ['-headless'],
-            },
+          ChromeHeadlessNoSandbox: {
+            base: 'ChromeHeadless',
+            flags: ['--no-sandbox'],
+          },
         },
 
-        browsers: ['ChromeHeadless'], // or 'Chrome'
+        browsers: ['ChromeHeadless', 'FirefoxHeadless'], // or 'Chrome'
```

Comment:
I'm reading this as this `customLaunchers.FirefoxHeadless` rm and `browsers.FirefoxHeadless` add is a refactor unrelated to getting Chrome to run un-sandboxed. (Nothing this in case it also helps anyone else understand this changeset. :) ).

Reply:
whoops forgot to copy over this comment https://github.com/uPortal-Project/uportal-home/pull/777#discussion_r163015010

Reply:
additionally amended commit with additional details.